test/math.test.ts

```diff
@@ -0,0 +1,13 @@
+import { clamp } from 'math';
+
+test('3 clamped by 1 & 2 should be 2', () => {
```

Comment:
these should all be in a `describe` block. Also, the test names should describe the particular feature, not the test case itself.